core/templates/dev/head/services/AssetsBackendApiService.js

```diff
@@ -19,37 +19,50 @@
 
 oppia.factory('AssetsBackendApiService', [
   '$http', '$q', 'UrlInterpolationService', 'AudioFileObjectFactory',
-  'FileDownloadRequestObjectFactory',
+  'ImageFileObjectFactory', 'FileDownloadRequestObjectFactory',
   function(
       $http, $q, UrlInterpolationService, AudioFileObjectFactory,
-      FileDownloadRequestObjectFactory) {
+      ImageFileObjectFactory, FileDownloadRequestObjectFactory) {
     // List of filenames that have been requested for but have
     // yet to return a response.
-    var _filesCurrentlyBeingRequested = [];
+    var _audioFilesCurrentlyBeingRequested = [];
+    var _imageFilesCurrentlyBeingRequested = [];
+
+    var AUDIO_ASSET = 'audio';
```

Comment:
Just a general note that the naming should be ASSET_TYPE_AUDIO, ASSET_TYPE_IMAGE. Basically, have the type of the thing come first, then the name of the thing itself. That way it's clear that all these fall under one category.

Reply:
Done!